kernel/lib/mutex.cpp

```diff
@@ -33,8 +33,8 @@
     }                           \
 }
 
-px_mutex::px_mutex() {
-    locked = false;
+px_mutex::px_mutex(const char *name) : locked(false) {
```

Comment:
Personally I prefer all of the property setters to be on a new line so that it's obvious they are getting set.

Reply:
Okay, I can do that.